src/components/demos/read/Demo.js

```diff
@@ -95,7 +97,22 @@ export default function ReadDemo(props) {
       <Header {...readDemo} />
       <Wrapper {...theme.wrapper}>
         {readDemo.showHackPopup && (
-          <ButtonSide {...readDemo} action={switchToMorello} />
+          <ButtonSide
+            {...readDemo}
+            learnMore={false}
+            action={switchToMorello}
+          />
+        )}
+        {renderExplainer && (
+          <ButtonSide
+            learnMore={true}
+            {...readDemo}
+            action={(e) => {
+              nav('/read-demo-explainer', { replace: false })
```

Comment:
`{ replace: false }` means we'll break back behaviour right? I'm not sure that's desired